### PR TITLE
[Do Not Merge] Merge dev into master Post release! 

### PR DESCRIPTION
**Pull request by Jaybsoni**
Prepared a PR which will merge new changes from the dev branch of the qml repo into master once all plugins and the new version of PL has been released!

### CODE REVIEW

File: demonstrations/braket-parallel-gradients.py [hunk 1]
```diff
@@ -9,7 +9,11 @@
 .. related::
 
     tutorial_qaoa_intro Intro to QAOA
+<<<<<<< HEAD
     vqe_parallel VQE with parallel QPUs on Rigetti Forest
+=======
+    vqe_parallel VQE with parallel QPUs with Rigetti
+>>>>>>> dev
```
Comment (josh146) [inline demonstrations/braket-parallel-gradients.py:16]:
```suggestion
    vqe_parallel VQE with parallel QPUs with Rigetti
```

File: demonstrations/tutorial_variational_classifier.py [hunk 2]
```diff
@@ -16,7 +16,11 @@
 
    tutorial_data_reuploading_classifier Data-reuploading classifier
    tutorial_multiclass_classification Multiclass margin classifier
+<<<<<<< HEAD
    ensemble_multi_qpu Ensemble classification with Forest and Qiskit devices
+=======
+   ensemble_multi_qpu Ensemble classification with Rigetti and Qiskit devices
+>>>>>>> dev
```
Comment (josh146) [inline demonstrations/tutorial_variational_classifier.py:23]:
```suggestion
   ensemble_multi_qpu Ensemble classification with Rigetti and Qiskit devices
```

File: demonstrations/tutorial_vqe.py [hunk 3]
```diff
@@ -10,7 +10,11 @@
 .. related::
 
    tutorial_quantum_chemistry Building molecular Hamiltonians
+<<<<<<< HEAD
    vqe_parallel VQE with parallel QPUs on Rigetti Forest
+=======
+   vqe_parallel VQE with parallel QPUs with Rigetti
+>>>>>>> dev
```
Comment (josh146) [inline demonstrations/tutorial_vqe.py:17]:
```suggestion
   vqe_parallel VQE with parallel QPUs with Rigetti
```

File: demonstrations/tutorial_vqe_spin_sectors.py [hunk 4]
```diff
@@ -8,7 +8,11 @@
 
 .. related::
    tutorial_vqe A brief overview of VQE
+<<<<<<< HEAD
    vqe_parallel VQE with parallel QPUs on Rigetti Forest
+=======
+   vqe_parallel VQE with parallel QPUs with Rigetti
+>>>>>>> dev
```
Comment (josh146) [inline demonstrations/tutorial_vqe_spin_sectors.py:15]:
```suggestion
   vqe_parallel VQE with parallel QPUs with Rigetti
```

File: demonstrations/tutorial_qubit_tapering.py [hunk 5]
```diff
@@ -14,9 +14,13 @@
     tutorial_adaptive_circuits Adaptive circuits for quantum chemistry
     tutorial_differentiable_HF Differentiable Hartree-Fock
 
+<<<<<<< HEAD
 
 *Author: Soran Jahangiri — Posted: 16 May 2022. Last updated: 24 May 2022.*
 
+=======
+*Authors: Utkarsh Azad and Soran Jahangiri. Posted: 16 May 2022. Last updated: 08 Nov 2022*
+>>>>>>> dev
```
Comment (josh146) [inline demonstrations/tutorial_qubit_tapering.py:23]:
```suggestion
*Authors: Utkarsh Azad and Soran Jahangiri. Posted: 16 May 2022. Last updated: 08 Nov 2022*
```

File: demonstrations/vqe_parallel.py [hunk 6]
```diff
@@ -12,13 +12,18 @@
 
    tutorial_vqe A brief overview of VQE
 
+<<<<<<< HEAD
 *Author: Tom Bromley — Posted: 14 February 2020. Last updated: 9 November 2022.*
+=======
+*Author: Tom Bromley — Posted: 14 February 2020. Last updated: 13 December 2021.*
+>>>>>>> dev
```
[the remark this answers is not in the document]
Comment (Jaybsoni) [reply demonstrations/vqe_parallel.py:19]: This should be 9 November 2022

**Review by josh146 — APPROVED** [review]

File: demonstrations/pytorch_noise.py [hunk 7]
```diff
@@ -13,28 +13,32 @@
 
    tutorial_noisy_circuit_optimization Optimizing noisy circuits with Cirq
 
+<<<<<<< HEAD
 *Author: Josh Izaac — Posted: 11 October 2019. Last updated: 9 November 2022.*
+=======
+*Author: Josh Izaac — Posted: 11 October 2019. Last updated: 14 September 2022.*
+>>>>>>> dev
```
Comment (josh146) [inline demonstrations/pytorch_noise.py:20]:
```suggestion
*Author: Josh Izaac — Posted: 11 October 2019. Last updated: 9 November 2022.*
```

File: demonstrations/vqe_parallel.py [hunk 8]
```diff
@@ -12,13 +12,14 @@
 
    tutorial_vqe A brief overview of VQE
 
-*Author: Tom Bromley — Posted: 14 February 2020. Last updated: 9 November 2022.*
+*Author: Tom Bromley — Posted: 14 February 2020. Last updated: 13 December 2021.*
```
Comment (josh146) [inline demonstrations/vqe_parallel.py:15]:
```suggestion
*Author: Tom Bromley — Posted: 14 February 2020. Last updated: 9 November 2022.*
```

File: demonstrations/pytorch_noise.py (hunk 7 above)
Comment (Jaybsoni) [inline demonstrations/pytorch_noise.py:20]:
```suggestion
*Author: Josh Izaac — Posted: 11 October 2019. Last updated: 9 November 2022.*
```

**Review by Jaybsoni — COMMENTED** [review]
Looks good to me 💯

**Pull request merged**